plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestPostgreSqlConnectorTest.java

```diff
@@ -204,6 +204,27 @@ public void testSystemTable()
                 .contains("orders");
     }
 
+    @Test
+    public void testPartitionedTables()
+            throws Exception
+    {
+        try {
+            String values202111 = "(1, 'A', '2021-11-01'), (2, 'B', '2021-11-25')";
+            String values202112 = "(3, 'C', '2021-12-01')";
+            execute("CREATE TABLE test_part_tbl (id int not null, payload varchar, logdate  date not null) PARTITION BY RANGE (logdate)");
+            execute("CREATE TABLE test_part_tbl_2021_11 PARTITION OF test_part_tbl FOR VALUES FROM ('2021-11-01') TO ('2021-12-01')");
+            execute("CREATE TABLE test_part_tbl_2021_12 PARTITION OF test_part_tbl FOR VALUES FROM ('2021-12-01') TO ('2022-01-01')");
+            execute(format("INSERT INTO test_part_tbl VALUES %s ,%s", values202111, values202112));
+            assertThat(computeActual("SHOW TABLES").getOnlyColumnAsSet())
+                    .contains("test_part_tbl", "test_part_tbl_2021_11", "test_part_tbl_2021_12");
+            assertQuery("SELECT * FROM test_part_tbl", format("VALUES %s, %s", values202111, values202112));
+            assertQuery("SELECT * FROM test_part_tbl_2021_12", "VALUES " + values202112);
+        }
+        finally {
+            execute("DROP TABLE IF EXISTS test_part_tbl");
```

Comment:
We should also test if drop table from Trino actually works as desired for partitioned tables too.

Reply:
Done in a second test table